docs/resources/vpcep_service.md

```diff
@@ -35,62 +35,70 @@ resource "flexibleengine_vpcep_service" "demo" {
     the VPC endpoint service belongs. Changing this creates a new VPC endpoint service.
 
-* `server_type` (Required) - Specifies the backend resource type. The value can be **VM**, **VIP** or **LB**.
+* `server_type` - (Required, String, ForceNew) Specifies the backend resource type. The value can be **VM**, **VIP** or **LB**.
     Changing this creates a new VPC endpoint service.
 
-* `port_id` (Required) - Specifies the ID for identifying the backend resource of the VPC endpoint service.
+* `port_id` (Required, String) - Specifies the ID for identifying the backend resource of the VPC endpoint service.
     - If the `server_type` is **VM**, the value is the NIC ID of the ECS where the VPC endpoint service is deployed.
     - If the `server_type` is **VIP**, the value is the NIC ID of the physical server where virtual resources are created.
     - If the `server_type` is **LB**, the value is the ID of the port bound to the private IP address of the load balancer.
 
-* `port_mapping` (Required) - Specified the port mappings opened to the VPC endpoint service.
+* `port_mapping` (Required, List) - Specified the port mappings opened to the VPC endpoint service.
     Structure is documented below.
 
-* `approval` (Optional) - Specifies whether connection approval is required. The default value is false.
+* `approval` (Optional, Bool) - Specifies whether connection approval is required. The default value is false.
 
-* `permissions` (Optional) - Specifies the list of accounts to access the VPC endpoint service.
+* `permissions` (Optional, Set) - Specifies the list of accounts to access the VPC endpoint service.
     The record is in the `iam:domain::domain_id` format, while `*` allows all users to access the VPC endpoint service.
 
-* `tags` - (Optional) The key/value pairs to associate with the VPC endpoint service.
+* `tags` - (Optional, Map) The key/value pairs to associate with the VPC endpoint service.
 
 The `port_mapping` block supports:
 
-* `protocol` - (Optional) Specifies the protocol used in port mappings.
+* `protocol` - (Optional, String) Specifies the protocol used in port mappings.
     The value can be *TCP* or *UDP*. The default value is *TCP*.
 
-* `service_port` - (Optional) Specifies the port for accessing the VPC endpoint service.
+* `service_port` - (Optional, Int) Specifies the port for accessing the VPC endpoint service.
     This port is provided by the backend service to provide services. The value ranges from 1 to 65535.
 
-* `terminal_port` - (Optional) Specifies the port for accessing the VPC endpoint.
+* `terminal_port` - (Optional, Int) Specifies the port for accessing the VPC endpoint.
     This port is provided by the VPC endpoint, allowing you to access the VPC endpoint service.
     The value ranges from 1 to 65535.
 
+* `service_type` - (Required, String, ForceNew)The type of the VPC endpoint service. Only **interface** can be configured.
```

Comment:
`service_type` is optional in schema, please check it carefully.

Reply:
revised